### DISCUSSION TITLE
labels not appearing at the correct time position

### DISCUSSION THREAD

**Discussion by CeliaHanna** [biigle/core#612] [General]
Recently when I started annotating a video on biigle, all of my annotations were being placed at the start of the video in the time bar, no matter where along the video I made them. This hasn't happened before and I have annotated many videos. Can anyone help explain / fix this issue, as I need accurate time stamps of my annotations for my project. Thanks

**Comment by mzur** [comment]
Hi, can you please send me an ID or link of the video so I can have a look? While only authorized users will be able to open the link, you can also send it to info@biigle.de if you don't want to disclose it here.
